api/src/main/java/org/openmrs/module/fgh/mpi/MpiConstants.java

```diff
@@ -164,4 +167,29 @@ public class MpiConstants {
 	
 	public final static String RESPONSE_FIELD_VALUE_REF = "valueReference";
 	
+	public final static String GP_SANTE_CLIENT_ID = MODULE_ID + ".sante.db.client.id";
+	
+	public final static String GP_SANTE_CLIENT_SECRET = MODULE_ID + ".sante.db.client.secret";
+	
+	public final static String GP_SANTE_LOGIN_TYPE = MODULE_ID + ".sante.db.login.type";
+	
+	public final static String GP_SANTE_SCOPE = MODULE_ID + ".sante.db.scope";
+	
+	public final static String GP_MPI_SYSTEM = MODULE_ID + ".system";
+	
+	public final static String GP_MPI_APP_CONTENT_TYPE = MODULE_ID + ".application.content.type";
+	
+	public final static String GP_AUTHENTICATION_TYPE = MODULE_ID + ".authentication.type";
+	
+	public final static String AUTHENTICATION_OAUTH_TYPE = "OAUTH";
+	
+	public final static String AUTHENTICATION_SSL_TYPE = "SSL";
```

Comment:
SSL is not authentication type, perhaps CERTIFICATE

Reply:
Did you see this comment?

Reply:
Yes